tailcfg/tailcfg.go

```diff
@@ -2653,6 +2653,17 @@ const (
 	// NodeAttrTrafficSteering configures the node to use the traffic
 	// steering subsystem for via routes. See tailscale/corp#29966.
 	NodeAttrTrafficSteering NodeCapability = "traffic-steering"
+
+	// NodeAttrTailnetDisplayName is how the tailnet name should be displayed to the user.
+	// Currently if this is set by a user this will only include letters, numbers,
+	// apostrophe, spaces, and hyphens. The only time this is not true,
+	// is if it defaults to the SID.
+	// i.e. "Tailscale - Dev", "User's Tailnet", or if it is a default SID
```

Comment:
Probably e.g. (example) instead of i.e. (that is), but if we were to follow Go style, we'd just avoid Latin-isms and write out "such as" or "like" or "for example".